uktrade/data-hub-frontend
# Only show aventri events to pilot group users

## Description of change

Only get run the query to get aventri ids if the feature flag is enabled for the logged in user

## Test instructions

- In the Admin site, set the `user-activity-stream-aventri` feature flag to be OFF for your user
- Go to the URL http://localhost:3000/companies/375094ac-f79a-43e5-9c88-059a7caa17f0/activity, you should not see any aventri events
- In the Admin site, set the `user-activity-stream-aventri` feature flag to be ON for your user
- Go to the URL http://localhost:3000/companies/375094ac-f79a-43e5-9c88-059a7caa17f0/activity, you should see aventri events

## Screenshots

### With feature flag

![image](https://user-images.githubusercontent.com/102232401/205385139-50a1713c-8872-4355-8093-d186d28a583d.png)

### Without feature flag

![image](https://user-images.githubusercontent.com/102232401/205385281-cbcae015-8e04-4c2b-b103-cd89dc747c69.png)

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
